contributed_definitions/nyaml/NXsts.yaml

```diff
@@ -294,14 +287,13 @@ NXsts(NXsensor_scan):
               This is set-point of tip current (in the constant current mode should be equal to set-point,
               in the constant height mode means the real tunnelling current between tip and sample).
           current_calibration(NX_NUMBER):
-            unit: NX_ANY
+            unit: NX_CURRENT
```

Comment:
If it's supposed to be `NX_CURRENT`, how does that work with the docstring that says it should be A/V?

Reply:
That was also in discussion with NAIC. Currently we do not have any compatible units to express A/V. Now, I would read it as value of current per unit. But if you have a suggestion, I can go with that.

Reply:
Then please mention this in the docstring.